### DISCUSSION TITLE
Docker image update

### DISCUSSION THREAD

**Discussion by robbrad** [jlesage/docker-crashplan-pro#467] [General]
Hi @jlesage 

Thank you so much for updating the repo and crashplan version - my crashplan is failing and the update is much needed

I noticed you commited yesterday but the docker image on docker IO remainted the same as the one from 7 months ago

https://hub.docker.com/r/jlesage/crashplan-pro/tags

Is this something that will change?
Thanks
Rob

**Comment by dcbasso** [comment]
If you need, you can use my image: dcbasso/crashplan-pro:1.0
I cant promess that will update, I create to myself and its the lastest version from this project.

**Comment by jlesage** [comment]
A new image is now available.

**Comment by robbrad** [comment]
thank you @jlesage
